docs/user_guide/plugin_development/messaging.rst

```diff
@@ -110,12 +106,12 @@ like so:
         @botcmd(template="hello")
         def hello(self, msg, args):
             """Say hello to someone"""
-            response = tenv().get_template('hello.html').render(name=args)
+            response = tenv().get_template('hello.md').render(name=args)
             self.send(msg.frm, response, message_type=msg.type)
 
 .. note::
     In both cases a plain-text version of your template is automatically
-    generated for clients and back-ends that do not support XHTML-IM.
+    generated for clients and back-ends that do not support Markdown.
```

Comment:
I think this note doesn't quite make sense anymore in this context. I think we should replace this with something along the lines of *"Not all networks support the full range of markup available with Markdown so backends convert the markup into a representation that is most suited for the network you are using. Because of this, you may find that your template is rendered differently from how you intended, or the template look different when compared between the various backends."*.